src/search-injection/components/Dropdown.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import styles from './Dropdown.css'
+
+const openSettings = () => {
+    const message = {
+        action: 'openOptionsURL',
```

Comment:
It would be good to define these action names somewhere which is imported by both this content script code and the corresponding listener in the background script code. Maybe the `search-injection/constants` module?

Reply:
Got it.